Formula/crc.rb

```diff
@@ -0,0 +1,28 @@
+class Crc < Formula
+  desc "OpenShift 4 cluster on your local machine"
+  homepage "https://code-ready.github.io/crc/"
+  url "https://github.com/code-ready/crc.git",
+      :tag      => "1.0.0",
+      :revision => "575079bff638214dc46d831caaef1c08927346ec"
+  head "https://github.com/code-ready/crc.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GO111MODULE"] = "on"
```

Comment:
```suggestion
```

We dont need this for go formula anymore.

Reply:
Fixed.